ehealthid-cli/src/main/java/com/oviva/ehealthid/cli/KeyGeneratorCommand.java

```diff
@@ -66,36 +57,18 @@ private String deriveName(URI issuer) {
     return s;
   }
 
-  private JWK generateSigningKey(URI issuer)
-      throws JOSEException, IOException, CertificateEncodingException, OperatorCreationException {
-
-    var key =
-        new ECKeyGenerator(Curve.P_256)
-            .keyUse(KeyUse.SIGNATURE)
-            .keyIDFromThumbprint(true)
-            .generate();
+  private JWK generateSigningKey() throws JOSEException {
 
+    // https://gemspec.gematik.de/docs/gemSpec/gemSpec_IDP_FD/gemSpec_IDP_FD_V1.7.2/#A_23185-01
     var now = Instant.now();
     var nbf = now.minus(Duration.ofHours(24));
-    var exp = now.plus(Duration.ofDays(180));
-
-    var cert =
```

Comment:
The signing key for the federation does not need a certificate, we can strip that here :)